.github/actions/setup-linux/action.yml

```diff
@@ -30,16 +30,21 @@ runs:
         fi
 
     - name: Log in to ECR
-      shell: bash
+      uses: nick-fields/retry@3e91a01664abd3c5cd539100d10d33b9c5b68482
       env:
         AWS_RETRY_MODE: standard
         AWS_MAX_ATTEMPTS: "5"
```

Comment:
These variables `AWS_RETRY_MODE` and `AWS_MAX_ATTEMPTS`.  I wonder if they do anything and could just be removed.  This code snippet looks weird, I'm counting several ways to retry that it refers to.

Reply:
These do indeed have an effect. See https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-retries.html

And yeah, we now have 3 levels of retries introduced here. One option is to take out the `retry ()` command and leave it with the nick-fields and `aws cli` retries